pkgs/tools/networking/dnsmasq/makefile-defaults.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/Makefile b/Makefile
+index 78e25f0..8e44354 100644
+--- a/Makefile
++++ b/Makefile
+@@ -18,31 +18,31 @@
+ 
+ # Variables you may well want to override.
+ 
+-PREFIX        = /usr/local
+-BINDIR        = $(PREFIX)/sbin
+-MANDIR        = $(PREFIX)/share/man
+-LOCALEDIR     = $(PREFIX)/share/locale
+-BUILDDIR      = $(SRC)
+-DESTDIR       = 
+-CFLAGS        = -Wall -W -O2
+-LDFLAGS       = 
+-COPTS         = 
+-RPM_OPT_FLAGS = 
+-LIBS          = 
++PREFIX        ?= /usr/local
++BINDIR        ?= $(PREFIX)/sbin
++MANDIR        ?= $(PREFIX)/share/man
++LOCALEDIR     ?= $(PREFIX)/share/locale
++BUILDDIR      ?= $(SRC)
++DESTDIR       ?=
++CFLAGS        ?= -Wall -W -O2
++LDFLAGS       ?=
++COPTS         ?=
++RPM_OPT_FLAGS ?=
++LIBS          ?=
+ 
+ #################################################################
+ 
+ # Variables you might want to override.
+ 
+-PKG_CONFIG = pkg-config
+-INSTALL    = install
+-MSGMERGE   = msgmerge
+-MSGFMT     = msgfmt
+-XGETTEXT   = xgettext
++PKG_CONFIG ?= pkg-config
++INSTALL    ?= install
++MSGMERGE   ?= msgmerge
++MSGFMT     ?= msgfmt
++XGETTEXT   ?= xgettext
+ 
+-SRC = src
+-PO  = po
+-MAN = man
++SRC ?= src
++PO  ?= po
++MAN ?= man
```

Comment:
Not sure if they want this on the dnsmasq list, but I assume it will be simple enough for the maintainers there to remove these 3 lines from the patch again.

Reply:
yeah i thought about that to. they scoped the vars in the section `Variables you might want to override` so i thought: let them decide

Reply:
Hmm http://lists.thekelleys.org.uk/pipermail/dnsmasq-discuss/ the message @betaboon's didn't seem to pop up yet. Maybe it is awaiting moderation?

Reply:
i sent it twice, tried to register twice nothing worked :/ i guess i have to contact simon kelly directly :/

Reply:
I was able to register, but no idea whether the helps with moderation or not.

Reply:
Yeah, I propose registering and re-sending - I assume the incoming message queue of unregistered mail adresses is not looked at:

> To control spam, only subscribers are allowed to post to the list.

Reply:
I heard scary things about dnsmasq upstream. Apparently the are pihole maintainer have better means to send fixes to dnsmasq upstream.